src/Mapper.cpp

```diff
@@ -303,3 +302,45 @@ void Mapper::countGates(decltype(qcMapped.cbegin())      it,
     }
   }
 }
+std::size_t Mapper::insertGate(const Edge& edge, qc::StandardOperation* op) {
+  qcMapped.emplace_back<qc::StandardOperation>(
+      qcMapped.getNqubits(), qc::Control{static_cast<qc::Qubit>(edge.first)},
+      edge.second, op->getType(), op->getParameter().at(0),
+      op->getParameter().at(1), op->getParameter().at(2));
+  return 1;
+}
+
+std::size_t Mapper::insertFlippedGate(const Edge&            edge,
+                                      qc::StandardOperation* op) {
+  int swapType = -1;
+  switch (op->getType()) {
+  case qc::X:
```

Comment:
```suggestion
  case qc::X:
  case qc::SX:
  case qc::SXdag:
```
I was able to verify these additional gates as well.